test/load/terraform/README.md

```diff
@@ -0,0 +1,30 @@
+## Terraform Consul Load Testing
+
+# How to use
+1. Build an image with the desired Consul version and a loadtest image in the Packer folder here.
+2. Create your own `vars.tfvars` file in this directory.
+3. Place the apprioate AMI IDs in the `consul_ami_id` and `test_server_ami` variables, here is an example of a `vars.tfvars`:
+```
+vpc_name             = "consul-test-vpc"
+vpc_cidr             = "11.0.0.0/16"
+public_subnet_cidrs  = ["11.0.1.0/24", "11.0.3.0/24"]
+private_subnet_cidrs = ["11.0.2.0/24"]
+vpc_az               = ["us-east-2a", "us-east-2b"]
+test_instance_type   = "t2.micro"
+test_server_ami      = "ami-0ad7711e837ebe166"
```

Comment:
Maybe add a comment here at the end of the line saying `# from the Packer build output`